src/js/utils/MarathonConfigUtil.js

```diff
@@ -0,0 +1,73 @@
+import HostUtil from '../utils/HostUtil';
+import Networking from '../constants/Networking';
+
+const serviceAddressKey = 'Service Address';
+
+function buildHostName(id, port) {
+  let hostname = HostUtil.stringToHostname(id);
+  return `${hostname}${Networking.L4LB_ADDRESS}:${port}`;
+}
+
+function defaultCreateLink(contents) {
+  return contents;
+}
+
+var MarathonConfigUtil = {
```

Comment:
Let's rename this into `ServiceConfigUtil` as discussed in: https://github.com/dcos/dcos-ui/pull/1118/files#r79790640